src/app/Fake.Core.UserInput/UserInput.fs

```diff
@@ -0,0 +1,63 @@
+namespace Fake.Core
+open System
+
+/// Helpers for capturing user input
+///
+/// ## Sample
+///
+///     UserInput.getUserInput prompt
+[<RequireQualifiedAccess>]
+module UserInput =
```

Comment:
This module is copied from an existing one from `FakeLib`, correct? Can we change the obsolete warnings of the existing module in `FakeLib` to point to this one?

Reply:
One extension to that: We also need to add this new module to `FakeLib` in order to enable the migration path.

Reply:
This should be addressed by linking the file in FakeLib which referenced by the Legacy-FAKE solution right?